ui/src/app/pages/data/data-set/genomic-extraction-modal.spec.tsx

```diff
@@ -145,10 +165,18 @@ describe('GenomicExtractionModal', () => {
       },
     ];
 
-    const wrapper = await component();
+    await componentAlt();
+    expect(
```

Comment:
Instead of testing for the absence of the data-test-id, testing for the absence of all three error messages that could be in that component.

Reply:
sounds like a good candidate for a const